Overview.html

```diff
@@ -4080,11 +4131,14 @@ <h3 id="response-class"><span class="secno">6.4 </span>Response class</h3>
     <p class="note no-backref"><code title="">101</code> is included in
     <a href="#null-body-status">null body status</a> due to its use elsewhere. It does not affect this step.
 
-   <li><p>Let <var title="">stream</var> and <var title="">Content-Type</var> be the result of
+   <li><p>Let <var title="">bytes</var> and <var title="">Content-Type</var> be the result of
    <a href="#concept-bodyinit-extract" title="concept-BodyInit-extract">extracting</a> <var title="">body</var>.
 
-   <li><p>Set <var title="">r</var>'s <a href="#concept-response-response" title="concept-Response-response">response</a>'s
-   <a href="#concept-response-body" title="concept-response-body">body</a> to <var title="">stream</var>.
+   <li><p>Set <var title="">r</var>'s
+   <a href="#concept-response-readable-stream" title="concept-Response-readable-stream">readable stream</a> to the result of
+   <a href="#concept-construct-fixed-readablestream" title="concept-construct-fixed-ReadableStream">constructing</a> a fixed
+   <a href="#concept-readablestream" title="concept-ReadableStream">ReadableStream</a> with an array consisting of a
```

Comment:
The input to the constructing a readable stream algorithm isn't a JS array, but instead an abstract list type. (In ES it would be a List, but most other specs including fetch are a bit less formal.) I'd say either "with a Uint8Array..." or "with a list whose single element is a Uint8Array..."